twitter_scraper/modules/profile.py

```diff
@@ -49,30 +50,41 @@ def __parse_profile(self, page):
             )
         except ParserError:
             pass
-
+        
+        # TODO unfixed
         try:
             self.is_private = html.find(".ProfileHeaderCard-badges .Icon--protected")[0]
             self.is_private = True
         except:
             self.is_private = False
 
+        # blue badge
+        self.is_verified = True
         try:
-            self.is_verified = html.find(".ProfileHeaderCard-badges .Icon--verified")[0]
-            self.is_verified = True
+            q = html.find("a.badge")[0]
+            if not q:
+                self.is_verified = False
         except:
             self.is_verified = False
 
-        self.location = html.find(".ProfileHeaderCard-locationText")[0].text
-        if not self.location:
+        try:
+            self.location = html.find('div.location')[0].text
+            if not self.location:
+                self.location = None
+        except:
             self.location = None
 
+        # TODO unfixed
         self.birthday = html.find(".ProfileHeaderCard-birthdateText")[0].text
```

Comment:
Not too familiar with the code (and appreciate it's not the issue you are fixing), but I would assume to make the code work it would be better to put this in a try except IndexError block?

Reply:
Hi, thank you for the response! 
The error is an error due to the empty list, I have added an exception catch for all elements, please try again.
I did not use specific IndexError is because the original code did not specify Errors, so I think it will be better to use the same style. 

Once again thank you for disclosing the problem!

Reply:
```
    profile = Profile(f"{user_id}")
    print(profile.to_dict())
```
You could try using the code by the fixed version above, I use "BarackObama" as user_id to gets the output below:
- output
```
{'name': 'Barack Obama', 'username': 'BarackObama', 'birthday': None, 'biography': 'Dad, husband, President, citizen.', 'location': 'Washington, DC', 'website': 'obama.org', 'profile_photo': 'https://pbs.twimg.com/profile_images/822547732376207360/5g0FC8XX_normal.jpg', 'banner_photo': None, 'likes_count': None, 'tweets_count': 15921, 'followers_count': 122418592, 'following_count': 601466, 'is_verified': True, 'is_private': False, 'user_id': None}
```

Reply:
New version is working! Thanks a lot!

Reply:
@yeachan153 could you please make a review confirm?

Reply:
> @yeachan153 Thank you too ~ could you please confirm the review request?

Seems valid. Could you please merge it? It's been quite a while, everything is better than the current version that is not working with profiles at all.